semgrep/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awaremonorepo.yaml-dependency_awaremonorepo/results.json

```diff
@@ -14,12 +14,12 @@
     {
       "check_id": "rules.dependency_aware.js-sca",
       "end": {
-        "col": 6,
-        "line": 1,
-        "offset": 5
+        "col": 0,
```

Comment:
@mmcqd looks like this test might not be deterministic?

Reply:
I'm going to merge since this sounds like it's unrelated to my PR